andela/noldor-ah-frontend
# #161290855 Codeclimate Setup

#### What does this PR do?
This PR integrates Codeclimate into the repo for automatic test reporting and code style checking.

#### Description of task to be completed
- add automatic test coverage reporting
- add automatic code linting check
- add codeclimate badge to README file

#### How should this be manually tested
- Go to the dashboard for this repo [here](https://codeclimate.com/github/andela/noldor-ah-frontend)
- You should see our test coverage and maintainability stats

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
#161290855

#### Screenshots
<img width="1089" alt="screen shot 2018-12-03 at 8 54 05 am" src="https://user-images.githubusercontent.com/36507972/49360286-0a8aba00-f6d9-11e8-815f-45c65d4bdf4e.png">
